golem_messages/datastructures/promissory.py

```diff
@@ -0,0 +1,128 @@
+import typing
+import uuid
+
+import eth_account.account
+import eth_account.messages
+
+from golem_messages.utils import uuid_to_bytes32
+
+PromissoryNoteSig = typing.NamedTuple(
+    'PromissoryNoteSig',
+    [
+        ('v', int),
+        ('r', bytes),
+        ('s', bytes),
+    ]
+)
+
+
+class PromissoryNote:
```

Comment:
Maybe it's a case of `datastructures.Container`?

Reply:
huh? it doesn't need to be serialized/deserialized...